app/forms/taxon_form.rb

```diff
@@ -1,5 +1,5 @@
 class TaxonForm
-  attr_accessor :title, :parent, :content_id, :base_path
+  attr_accessor :title, :taxon_parents, :content_id, :base_path
```

Comment:
It's a good point. What we wanted to communicate here is that this taxon has a list of parents (multiple ones, not just one) that are all taxons too.

The reason for this name here was to follow a bit what comes from the schemas (PRL: https://github.com/alphagov/govuk-content-schemas/pull/355/files) so we have consistency in the naming. However, I do think that `parent_taxons` sounds better...

I will change this both here and in the schema. 👍